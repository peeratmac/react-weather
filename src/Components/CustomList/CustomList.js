import React, { Component } from 'react';

class CustomList extends Component {
  render() {
    return (
      <div>
        <h2></h2>
      </div>
    );
  }
}

export default CustomList;
