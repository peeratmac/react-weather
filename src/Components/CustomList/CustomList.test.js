import React from 'react';
import { shallow } from 'enzyme';
import { CustomList } from './CustomList';

describe('CUSTOM LIST', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <CustomList cities={''} stationIDs={[]} weatherInfo={[]} />
    );
  });

  it('should match the CustomList Snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
