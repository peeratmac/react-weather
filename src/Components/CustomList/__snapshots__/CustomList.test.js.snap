// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CUSTOM LIST should match the CustomList Snapshot 1`] = `null`;
